src/AgGrid.elm

```diff
@@ -620,6 +729,43 @@ columnDefEncoder gridConfig columnDef =
                     columnDef.settings.suppressSizeToFit
           )
         , ( "suppressMenu", Json.Encode.bool columnDef.settings.suppressMenu )
+        , ( "valueFormatter"
+          , case ( columnDef.settings.valueFormatter, columnDef.renderer ) of
+                ( Just overwrittenFormatter, _ ) ->
```

Comment:
I am not sure I like this behavior (actually not sure). Maybe it would be better to provide different functions for default settings?